addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -48,6 +48,8 @@
             </t>
             <div class="container o_wslides_lesson_main">
                 <div class="row">
+                    <t t-set="can_access_channel" t-value="slide.channel_id.is_member or slide.channel_id.can_publish"/>
+                    <t t-set="default_background_image" t-value="(website.image_url(slide.channel_id, 'image_256') if slide.channel_id.image_256 else '/website_slides/static/src/img/channel-%s-default.jpg' % ('training' if slide.channel_id.channel_type == 'training' else 'documentation'))"/>
```

Comment:
Isn't 256 a bit tiny as background?

Maybe it's a background to a very small element though, hard to say just reading the XML.